### PR TITLE
[WIP] Nodejs Jenkins Agent upgrade release note 3.11

### PR DESCRIPTION
**Pull request by codyhoag**
This is a sample for the release note we'll add for the 3.11 z-stream release describing the upcoming additional two Nodejs Jenkins Agent images. This PR is for review purposes only.

Preview: http://file.rdu.redhat.com/~choag/052720/nodejs-image-test/release_notes/ocp_3_11_release_notes.html#ocp-3-11-x-added-nodejs-jenkins-agent-v10-and-v12

### CODE REVIEW

**Comment by codyhoag** [comment]
@akram can you review this 3.11 edition for accuracy? Thanks!

**Comment by codyhoag** [comment]
@sferich888 PTAL? This is intended for 3.11 only.

**Comment by akram** [comment]
That's perfect:
/lgtm

File: release_notes/ocp_3_11_release_notes.adoc [hunk 1]
```diff
@@ -3233,3 +3233,49 @@ link:https://access.redhat.com/errata/RHBA-2020:1551[RHBA-2020:1551] advisory.
...
 methods and strategies] for instructions.
+
+[[ocp-3-11-x-features]]
+==== Features
+
+[[ocp-3-11-x-added-nodejs-jenkins-agent-v10-and-v12]]
+===== Added Node.js Jenkins Agent v10 and v12
+
+The `jenkins-agent-nodejs-10-rhel7` and `jenkins-agent-nodejs-12-rhel7` images
+have been added to {product-title}. These new images allow Jenkins Pipelines to
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc:3244]: s/have been added/are now added

File: release_notes/ocp_3_11_release_notes.adoc [hunk 2]
```diff
@@ -3233,3 +3233,49 @@ link:https://access.redhat.com/errata/RHBA-2020:1551[RHBA-2020:1551] advisory.
...
+be upgraded to use either v10 or v12 of the Node.js Jenkins Agent. The Node.js
+v8 Jenkins Agent has been deprecated, but will continue being provided. For
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc:3246]: s/has been/is now

Comment (ahardin-rh) [reply release_notes/ocp_3_11_release_notes.adoc:3246]: s/being provided/ to be provided

**Comment by ahardin-rh** [comment]
@codyhoag Just some minor comments from me. Overall, looks good!

**Comment by codyhoag** [comment]
Thanks all! Docs have been approved and are now ready. :+1: 

These edits will be applied to the 3.11.z release Errata when the new Node.js images are made available. This PR will serve as the review for that future PR.

**Pull request closed without merging**